frontend/src/pages/scenes/Scene.tsx

```diff
@@ -188,14 +187,14 @@ const SceneComponent: FC<Props> = ({ scene }) => {
             </div>
             <hr />
             <div>
-              <strong className="me-2">Studio URL: </strong>
-              <a
-                href={getUrlByType(scene.urls, "STUDIO")}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {getUrlByType(scene.urls, "STUDIO")}
-              </a>
+              {scene.urls.map((u) => (
+                <>
+                  <strong className="me-2">{u.site?.name ?? "Unknown"}:</strong>
+                  <a href={u.url} target="_blank" rel="noopener noreferrer">
+                    {u.url}
+                  </a>
+                </>
+              ))}
```

Comment:
This is showing all the links. Only difference between it and the links tab is the site name vs the site icon.
While I like the description tab listing the studio URL, I don't think showing all the URLs is desirable.